toastify/toastify.nuspec

```diff
@@ -4,9 +4,9 @@
   <metadata>
    <id>toastify</id>
    <title>toastify</title>
-   <version>1.7.8</version>
+   <version>1.8.3</version>
    <authors>Geoffrey Huntley</authors>
-   <owners>ghuntley</owners>
+   <owners>ghuntley, slayerice09</owners>
```

Comment:
Alright, I've spoken with one of the chocolatey maintainers - they didn't follow process properly and added you as a maintainer without contacting myself which caught me off-guard. My qualm is with Chocolatey, not you. Happy to have you as a maintainer.